graphene_django_plus/mutations.py

```diff
@@ -547,11 +571,21 @@ def perform_mutation(cls, root, info, **data):
         cls.save(info, instance, cleaned_input)
 
         # save m2m data
-        for f in itertools.chain(instance._meta.many_to_many,
-                                 instance._meta.private_fields):
+        for f in itertools.chain(
+            instance._meta.many_to_many,
+            instance._meta.related_objects,
+            instance._meta.private_fields,
+        ):
+
+            if isinstance(f, (ManyToOneRel, ManyToManyRel)):
+                # Handle reverse side relationships.
+                input_val = cleaned_input.get(f.related_name, None)
+                target_field = getattr(instance, f.related_name)
+                target_field.set(input_val or [])
+
             if not hasattr(f, 'save_form_data'):
                 continue
-
+            
```

Comment:
be sure to run `flake8` to catch those issues

Reply:
thanks @bellini666 - I ran it and then made some changes and forgot to run it before the commit. I will add a commit hook as well.

Reply:
Now done! I have added a sensible pre-commit yaml file which takes care of all the styling issues I had left behind.